app/src/main/res/values-it/strings.xml

```diff
@@ -262,4 +297,44 @@
     <string name="time_ago_ago">%1$s %2$s fa</string>
     <string name="topAppBar_back">Indietro</string>
     <string name="utils_login_first">Effettua prima il login</string>
+    <string name="upvote">Voto positivo</string>
+    <string name="downvote">Voto negativo</string>
+    <string name="upvoted">Votato positivamente</string>
+    <string name="downvoted">Votato negativamente</string>
+    <string name="saved_image">Immagine salvata</string>
+    <string name="saving_image">Salvando l\'immagine…</string>
+    <string name="permission_denied">Permesso negato</string>
+    <string name="inbox_activity_replies">Risposte</string>
+    <string name="inbox_activity_mentions">Menzioni</string>
+    <string name="inbox_activity_messages">Messaggi</string>
+    <string name="bookmarks_activity_saved">Salvati</string>
+    <string name="generic_error">Qualcosa è andato storto</string>
+    <string name="create_edit_post_close">Chiudi</string>
+    <string name="lang_language">Lingua</string>
+    <string name="loading">Caricamento</string>
+    <string name="blur_nsfw">Sfoca immagini NSFW</string>
+    <string name="sorttype_tophour">Top dell\'ora</string>
+    <string name="sorttype_topsixhour">Top di 6 ore</string>
+    <string name="sorttype_toptwelvehour">Top di 12 ore</string>
+    <string name="sorttype_topthreemonths">Top di 3 mesi</string>
+    <string name="sorttype_topsixmonths">Top di 6 mesi</string>
+    <string name="sorttype_topninemonths">Top di 9 mesi</string>
+    <string name="dialogs_top_hour">Top dell\'ora</string>
+    <string name="dialogs_top_six_hour">Top di 6 ore</string>
+    <string name="dialogs_top_twelve_hour">Top di 12 ore</string>
+    <string name="dialogs_top_three_month">Top di 3 mesi</string>
+    <string name="dialogs_top_six_month">Top di 6 mesi</string>
+    <string name="dialogs_top_nine_month">Top di 9 mesi</string>
+    <string name="post_listing_view_source">Mostra markdown</string>
+    <string name="post_listing_view_original">Mostra originale</string>
+    <string name="look_and_feel_show_text_descriptions_in_navbar">Mostra descrizioni nella barra di navigazione</string>
+    <string name="person_iconBot">È un bot</string>
+    <string name="download">Download</string>
+    <string name="back_confirmation">Premi indietro nuovamente per uscire</string>
+    <string name="no_back_confirmation">Non chiedere conferma per uscire</string>
+    <string name="press_again_confirmation">Premi indietro due volte per uscire</string>
+    <string name="confirm_exit">Conferma azione di uscita</string>
+    <string name="back_dialog">Mostra un dialogo di conferma per uscire</string>
```

Comment:
I think that the correct translation for a dialog would be the somewhat verbose `finestra di dialogo`
What about replacing `dialogo` with `popup`, instead?
Or, more freely, `Richiedi conferma per uscire`?

Reply:
I like `Richiedi conferma per uscire`, done